packages/core/src/dialects/db2/query-generator-typescript.ts

```diff
@@ -213,9 +209,9 @@ export class Db2QueryGeneratorTypeScript extends AbstractQueryGenerator {
     if (options) {
       rejectInvalidOptions(
         'removeIndexQuery',
-        this.dialect.name,
+        this.dialect,
         REMOVE_INDEX_QUERY_SUPPORTABLE_OPTIONS,
-        REMOVE_INDEX_QUERY_SUPPORTED_OPTIONS,
+        EMPTY_ARRAY,
```

Comment:
This should also be an empty set right? And then the added import can also be removed.
Should this be picked up by the typings?

Reply:
Oh, I actually made it accept any iterable so we don't even need to use EMPTY_SET in any of these

It doesn't change much, and it will be converted to a set in every case, which one should we opt for?

Reply:
I think for consistency and limiting what will be importing we should use EMPTY_SET in this case, but I'm fine with accepting any iterable. No need to update the function itself

Reply:
Done!

The function should accept anything, it's all normalized & cached anyway :)